### PR TITLE
RavenDB-11873 New database from v3.x usability issues

### PR DESCRIPTION
**Pull request by ml054**

### CODE REVIEW

File: src/Raven.Client/ServerWide/Operations/Migration/OfflineMigrationConfiguration.cs [hunk 1]
```diff
@@ -8,6 +8,10 @@ namespace Raven.Client.ServerWide.Operations.Migration
 {
     public class OfflineMigrationConfiguration
     {
+        private const string StorageExporterExecutable = "Raven.StorageExporter.exe";
+        private const string EsentDataFile = "Data.jfm";
```
Comment (ayende) [inline src/Raven.Client/ServerWide/Operations/Migration/OfflineMigrationConfiguration.cs]: The Esent filename is supposed to be just `Data`, no?

Comment (ml054) [reply src/Raven.Client/ServerWide/Operations/Migration/OfflineMigrationConfiguration.cs]: image

I can see data and data.jfm. I used data.jfm as it is has better selectivity then simply *data*. Anyway I can change that if you wish.

Comment (ml054) [reply src/Raven.Client/ServerWide/Operations/Migration/OfflineMigrationConfiguration.cs]: I just saw conversation on youtrack. I will change this to Data or Data.ravenfs

Comment (ml054) [reply src/Raven.Client/ServerWide/Operations/Migration/OfflineMigrationConfiguration.cs]: Fixed

File: src/Raven.Server/Web/Studio/StudioTasksHandler.cs [hunk 2]
```diff
@@ -56,6 +57,55 @@ public Task FullDataDirectory()
             return Task.CompletedTask;
         }
 
+        [RavenAction("/studio-tasks/offline-migration-test", "GET", AuthorizationStatus.Operator)]
+        public Task OfflineMigrationTest()
+        {
```
Comment (ppekrol) [inline src/Raven.Server/Web/Studio/StudioTasksHandler.cs:62]: I think we should throw here when server is on non-Windows?

File: src/Raven.Server/Web/System/AdminDatabasesHandler.cs [hunk 3]
```diff
@@ -1036,13 +1036,10 @@ public async Task MigrateDatabaseOffline()
             }
 
             var dataDir = configuration.DataDirectory;
-            if (Directory.Exists(dataDir) == false)
```
Comment (ppekrol) [inline src/Raven.Server/Web/System/AdminDatabasesHandler.cs:1039]: This method should also throw for non-Windows server. UI in Studio should be blocked

**Review by ppekrol — CHANGES_REQUESTED** [review]

**Comment by ml054** [comment]
I've blocked this on non-Windows

**Comment by ml054** [comment]
fixed url conventions

**Pull request merged**